### DISCUSSION TITLE
It would be great if Bun support linting and formatting out of the box.

### DISCUSSION THREAD

**Discussion by adeel0x01** [oven-sh/bun#5834] [Ideas]
### What is the problem this feature would solve?

Currently, users who work with Bun need to rely on external tools and configurations to ensure code quality and consistency, which can be cumbersome and time-consuming. This feature would streamline the development process and enhance the overall developer experience by integrating linting and formatting directly into Bun.

### What is the feature you are proposing to solve the problem?

The proposed feature is to incorporate built-in linting and formatting support into Bun. Users would have the flexibility to customize linting and formatting rules to suit their specific project requirements.

**Comment by sirenkovladd** [comment]
Duplicate of https://github.com/oven-sh/bun/discussions/712
